systemd/jenkins.conf

```diff
@@ -0,0 +1 @@
+D /run/jenkins 0770 jenkins jenkins -
```

Comment:
`RuntimeDirectory="jenkins"`

in `jenkins.service` should be a better fit here than a tmpfile.

Reply:
> in `jenkins.service` should be a better fit here than a tmpfile.

It is usually much more effective to submit a pull request  with an explanation why a different location would be better.  If you're not ready to submit a pull request, then an issue report is another way to start the conversation.

Commenting on a merged pull request is not nearly as likely to result in any action.